apps/web-giddh/src/app/vat-report/vat-report.component.ts

```diff
@@ -123,7 +131,11 @@ export class VatReportComponent implements OnInit, OnDestroy {
         { label: 'EUR', value: 'EUR', additional: { symbol: '€' } }
     ];
     /** Holds Current Currency Code for Zimbabwe report */
-    public vatReportCurrency: 'BWP' | 'USD' | 'GBP' | 'INR' | 'EUR' = this.vatReportCurrencyList[0].value;
+    public vatReportCurrencyCode: 'BWP' | 'USD' | 'GBP' | 'INR' | 'EUR' = this.vatReportCurrencyList[0].value;
+    /** Holds Current Currency Symbol for Zimbabwe report */
+    public vatReportCurrencySymbol: string = this.vatReportCurrencyList[0].additional.symbol;
```

Comment:
```suggestion
    public vatReportCurrencySymbol: string = this.vatReportCurrencyList[0]?.additional?.symbol;
```

Reply:
No need, It always get additional and symbol.